### ISSUE TITLE
Increase our reach by adding CLI help page to `tldr-pages/tldr`

### ISSUE THREAD

**Issue by andy1li** [codecrafters-io/cli#37]
Re: [Help Scout #4871](https://secure.helpscout.net/conversation/2682293623/4871?folderId=4697252)

You probably already know it, but for the sake of context: 

[tldr-pages/tldr](https://github.com/tldr-pages/tldr) is the most popular alternative to `man` pages. A lot of people prefer it because it's succinct and looks way nicer.

@rohitpaulk  Should we add a page for our CLI to it? And its priority level?

---

tldr looks like this:

<img width="753" alt="image" src="https://github.com/user-attachments/assets/9a473950-a9c9-4045-8789-ff6e3b84d058">

**Comment by andy1li** [comment]
Moved to Linear.
